x-pack/plugins/upgrade_assistant/server/lib/es_deprecations_status.ts

```diff
@@ -18,10 +18,24 @@ import {
 export function getShardCapacityDeprecationInfo({
   symptom,
   details,
+  status,
 }: {
   details: any;
   symptom: any;
+  status: estypes.HealthReportIndicatorHealthStatus;
```

Comment:
Thanks for addressing my feedback! Now that we don't use `status`, I think we could remove this parameter.

Otherwise, code changes LGTM.